endpoints/info/bidders.go

```diff
@@ -8,9 +8,11 @@ import (
 	"github.com/golang/glog"
 	"github.com/julienschmidt/httprouter"
 	"github.com/prebid/prebid-server/openrtb_ext"
-	yaml "gopkg.in/yaml.v2"
+	"gopkg.in/yaml.v2"
 )
 
+var responses map[string]json.RawMessage
```

Comment:
Agreed. The thought hit me about the  `NewBidderDetailsEnpdoint()` but still went for it because it is definitely called on startup and definitely throws an error on startup if anything was wrong. There is no chance that `NewBidderDetailsEndpoint` is executed and `responses` is nil. So don't think it should break.

But frankly, that was a half-hearted implementation. Appreciate your suggestion. I'll try it out.